run-nextflow-tests/configtest.py

```diff
@@ -18,6 +18,9 @@
 T = TypeVar('T', bound='NextflowConfigTest')
 
 
+BAD_CHARACTERS = re.compile(r'[":<>|*?\r\n\\/]')
```

Comment:
I suggest whitelisting and restricting to a list of characters we know are safe.

Reply:
This list comes from the upload-artifacts action: https://github.com/actions/upload-artifact/issues/333

> Invalid characters include:  Double quote ", Colon :, Less than <, Greater than >, Vertical bar |, Asterisk *, Question mark ?, Carriage return \r, Line feed \n

I didn't open that issue, but I do recall getting a similar warning. I also added `/` as a disallowed character.

Reply:
Okay. I'm okay to leave it for now, though generally having a whitelist is safer as it doesn't require chasing down all possible problematic characters by just blanket disallowing anything not expected.